controllers/sink.go

```diff
@@ -49,35 +45,36 @@ func (r *SinkReconciler) ObserveSinkStatefulSet(ctx context.Context, req ctrl.Re
 	}, statefulSet)
 	if err != nil {
 		if errors.IsNotFound(err) {
-			r.Log.Info("sink is not ready yet...")
+			r.Log.Info("sink statefulset is not found...")
+			condition.Action = v1alpha1.Create
+			sink.Status.Conditions[v1alpha1.StatefulSet] = condition
 			return nil
 		}
-		return err
-	}
 
-	selector, err := metav1.LabelSelectorAsSelector(statefulSet.Spec.Selector)
-	if err != nil {
-		r.Log.Error(err, "error retrieving statefulSet selector")
+		sink.Status.Conditions[v1alpha1.StatefulSet] = condition
 		return err
+	} else {
```

Comment:
the `golint` has some complain. could you fix it?

Reply:
fixed